test/sql/json/scalar/test_json_pretty_print.test

```diff
@@ -0,0 +1,39 @@
+# name: test/sql/json/scalar/test_json_pretty_print.test
+# description: Test JSON pretty printer
+# group: [scalar]
+
+require json
+
+statement ok
+pragma enable_verification
+
+statement ok
+create table test (j varchar)
+
+statement ok
+insert into test values ('{}'), ('[]'), ('[1, 2, 3]')
+
+mode output_hash
```

Comment:
Was this left in by accident?

Reply:
Yes, I removed the test table now.

Reply:
I meant the `mode output_hash`, not the test table. Adding `mode output_hash` prevents the test from actually testing anything